lib/timeutils/misc.c

```diff
@@ -142,3 +144,23 @@ timespec_diff_nsec(struct timespec *t1, struct timespec *t2)
 {
   return (glong)((t1->tv_sec - t2->tv_sec) * 1e9) + (t1->tv_nsec - t2->tv_nsec);
 }
+
+void timeutils_setup_timezone_hook(void);
+
+static void timeutils_reset_timezone(gint type, gpointer context)
+{
```

Comment:
please break the return value into a separate line as elsewhere in the codebase.